andela/wger-space
# #166185767 Change Workoutlog and WorkoutSession Relations

#### What does this PR do?

Changes the relations for Workoutlog and WorkoutSession relationship from common dates to a foreign key.

#### Description of Task to be completed?

When the dates for the same models happen to be different, this may cause misinformation in the logs, so I added a foreign key referencing the Logs to help keep both models references at per at all times.

#### How should this be manually tested?
Look at the updated tests that were written for this implementation before that should be passing as before.
- pull this branch locally checkout to it.
- run ```python manage.py makemigrations``` and ```python manage.py migrate```
- run ```python manage.py test wger.manager.tests.test_weight_log.WeightlogTestCase```
- The test should pass as before

#### Any background context you want to provide?

Due to the back end nature of this feature, testability will basically be based on tests passing functionality.

#### What are the relevant pivotal tracker stories?
[#166185767](https://www.pivotaltracker.com/story/show/166185767)
